jsk_vil_ros/node_scripts/ofa.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python
+import actionlib
+import base64
+import cv2
+from cv_bridge import CvBridge
+from dynamic_reconfigure.server import Server
+from jsk_vil_msgs.cfg import OFAQueryConfig
+from jsk_vil_msgs.msg import QuestionAndAnswerText
+from jsk_vil_msgs.msg import VQATaskAction
+from jsk_vil_msgs.msg import VQATaskFeedback
+from jsk_vil_msgs.msg import VQATaskResult
+from jsk_vil_msgs.msg import VQAResult
+import json
+import requests
+from requests.exceptions import ConnectionError
+import rospy
+from sensor_msgs.msg import Image
+from std_msgs.msg import String
+
+bridge = CvBridge()
+
+def ros_img_to_base(ros_img):
+    cv_img = bridge.imgmsg_to_cv2(ros_img, desired_encoding="bgr8")
+    # convert to base64
+    encimg = cv2.imencode(".png", cv_img)[1]
+    img_str = encimg.tostring()
+    img_byte = base64.b64encode(img_str).decode("utf-8")
+    return img_byte
+
+class OfaJSKROSClient:
```

Comment:
```suggestion
class OFAClientNode(object):
```

the program name should be `ofa_client_node.py`